docs/vertiq_testing_tool/vertiq_testing_tool_start_guide.rst

```diff
@@ -0,0 +1,128 @@
+
+#. The :green:`Plotting` section contains a live plot in the upper section and a dashboard with widgets to control the live plot in the lower section.
+
+#. The :purple:`Control Panel` section contains widgets to configure specific parameters and to control your module. The Control Panel on each tab contains different widgets.
+
+In the ESC Input tab shown above, the Control Panel contains the same parameters necessary to configure your module for use with a flight controller as in this :ref:`Control Center example <flight_controller_config_with_control_center>`.
+
+
+==================
+Continuous command
+==================
+
+.. image:: ../_static/vertiq_testing_tool_pictures/esc_input_command_widget.png
+
+A notable feature in the ESC Input tab is the 'Continuous command' toggle at the bottom of the Control Panel. This feature is disabled by default, which means you need to click the 'Command ESC' button every time you want to send an ESC command.
+When this feature is enabled, the ESC command will be continuously sent at the 'Command rate' with 10 Hz being the default value. The 'Command rate' can be adjusted to send the ESC command at different rates. 
+You can also use the slider to adjust the ESC input value, which mimics adjusting the flight controller's throttle input between [0, 1].
+
+
+================================
+Live Plotting and Active Metrics
+================================
+
+.. image:: ../_static/vertiq_testing_tool_pictures/voltage_input_command_example.png
+One main feature of Vertiq Testing Tool is the ability to plot any supported metric. 
+Please refer to the :ref:`IQUART Client Table Reference tables <iquart_client_reference_tables>` to find the endpoints that allows for 'get' in the Access field, 
+as these are the supported metrics that can be plotted.
+
+For example, let's command the module to a 3V drive voltage using the Control Panel in the Voltage Input tab and plot the *drive_volts* from the Brushless Drive client.
+
+.. note::
+    Please refer to the :ref:`Velocity and Voltage Based Control Mechanisms <manual_velocity_control_mechanisms>` documentation to learn more about Velocity and Voltage Control.
+
+.. warning::
+    Please remove all propellers from any module you plan on testing. Failure to do so can result in harm to you or others around you. Further, please ensure that your module is secured to a stationary platform or surface before attempting to spin it. 
+
+Set the 'Target Voltage' to 3.00 and click 'Send Command'. The module will ramp up to 3 volts over the next 5 seconds specified by the 'Ramp Time' value.
+After 5 seconds, the module should now have a constant, 3V, drive voltage. We can confirm this by plotting the drive volts.
+
+.. image:: ../_static/vertiq_testing_tool_pictures/drive_volts_metric_plotting_example.png
```

Comment:
more of a curiosity thing. It seems like this doesn't really "ramp" from 0 to 3V over 5 seconds. It seems like a step form 0 to 2.4V, then ramp for some amount of time to 3V. Does the ramp step up like that in actuality? If not, it'd be nice to see a plot of the motor ramping from 0 to 3V in a nice linear path if that's what's meant to happen.